OrbitGl/App.cpp

```diff
@@ -771,29 +771,60 @@ void OrbitApp::LoadModuleOnRemote(int32_t process_id,
                                   const std::shared_ptr<Module>& module,
                                   const std::shared_ptr<Preset>& preset) {
   thread_pool_->Schedule([this, process_id, module, preset]() {
-    const auto remote_symbols_result =
-        process_manager_->LoadSymbols(module->m_FullName);
-
-    if (!remote_symbols_result) {
-      SendErrorToUi(
-          "Error loading symbols",
-          absl::StrFormat(
-              "Did not find symbols on local machine for module \"%s\".\n"
-              "Trying to load symbols from remote resulted in error "
-              "message: %s",
-              module->m_Name, remote_symbols_result.error().message()));
+    const std::string& module_path = module->m_FullName;
+    const std::string& build_id = module->m_DebugSignature;
```

Comment:
I think `module->m_DebugSignature` can be empty here. This is not an issue here, but might be one later

Reply:
I added handling of empty build id on the service side - it should correctly check/disregard it now. It disregards it only if build_id is empty and we ended up finding symbols in the original file.